server/repo-mgr.h

```diff
@@ -7,6 +7,9 @@
 #include "commit-mgr.h"
 #include "branch-mgr.h"
 
+#define REPO_STATUS_NORMAL 0
+#define REPO_STATUS_READ_ONLY 1
```

Comment:
An enum should be defined. The last enum entry should be N_REPO_STATUS = 2. This way, even if in the future new repo status is added, other code can still use this symbol to refer to the number of status without change.